conda/install.py

```diff
@@ -225,21 +224,6 @@ def warn_failed_remove(function, path, exc_info):
         log.warn("Cannot remove, unknown reason: {0}".format(path))
 
 
-def exp_backoff_fn(fn, *args):
-    """Mostly for retrying file operations that fail on Windows due to virus scanners"""
-    max_retries = 5
-    for n in range(max_retries):
-        try:
-            result = fn(*args)
-        except Exception as e:
-            log.debug(repr(e))
-            if n == max_retries-1:
-                raise
-            time.sleep((2 ** n) + (random.randint(0, 1000) / 1000))
```

Comment:
Why not simply define `exp_backoff_fn` (and the `random` import) on Windows only.